flink/src/main/java/org/apache/iceberg/flink/sink/BaseDeltaTaskWriter.java

```diff
@@ -63,6 +70,15 @@ RowDataWrapper wrapper() {
     return wrapper;
   }
 
+  RowData projectDeleteData(RowData data) {
+    wrapper.wrap(data);
```

Comment:
should this be `deleteWrapper`?

Reply:
this is to wrap the raw data ，I use RowDataWrapper to select the primary key to generate the projected RowData.